src/utilities/FormManager.ts

```diff
@@ -108,26 +104,30 @@ class FormManager {
     this.notifyUserAboutErrors(errorMessages);
   }
 
-  public async onSave (event: any) {
-    const { form, onSave } = this.args;
+  private async validateThenSaveCallback (errors: any, _values: any) {
+      const { onSave } = this.args;
+      this.saving = true;
+      if (errors) { this.saving = false; return; }
```

Comment:
Would `this.saving` profit from being a `SmartBool` or not really ? 🤔

Reply:
I thought about it before, but the places it was being used were all too complex for a simple `.until(`. This seems to fit that use-case as well.